src/main/kotlin/com/terraformation/backend/funder/db/FundingEntityStore.kt

```diff
@@ -63,8 +64,11 @@ class FundingEntityStore(
                   groupRecords
                       .filter { it[PROJECTS.ID] != null }
                       .map { record ->
-                        SimpleProjectModel(
-                            id = record[PROJECTS.ID]!!, name = record[PROJECTS.NAME]!!)
+                        ProjectModel(
+                            id = record[PROJECTS.ID]!!,
+                            name = record[PROJECTS.NAME]!!,
+                            organizationId = record[PROJECTS.ORGANIZATION_ID]!!,
+                        )
```

Comment:
Could use `ProjectModel.of(record)`.

Reply:
Scratch this -- that'll throw an exception.

Reply:
Just to confirm my understanding, that would throw an exception because my query is only requesting id, name and orgId?